lib/screens/clock_screen.dart

```diff
@@ -90,7 +94,7 @@ class _ClockScreenState extends State<ClockScreen> {
               height: 100.0,
```

Comment:
use paddings/margins/ relative size instead of hardcoded width/height 
**because UI could SUCK** on other devices with different screen metrics